docs/change-log.md

```diff
@@ -2,6 +2,21 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update March 22nd, 2024
+
+- **Improved** The `artefact` trigger in Humanitec Pipelines now enforces serial execution, so a batch will be kept open until the previous batch has finished running. No explicit concurrency-group setting is required.
```

Comment:
@astromechza please let me know if this is user-facing improvement or an implementation one, or how else I could describe that update in a way users would understand, like pipelines running in parallel without explicit configuration?